README.rst

```diff
@@ -692,7 +700,7 @@ variable ``FUN``:
         argument ``append_suffix`` can be passed at the end to control whether
         ``VAR.suffix`` is appended to each destination file. ``append_suffix``
         defaults to ``True`` if not given.
-``desc(OPTS, [type[, prolog[, epilog]]])``
+``desc([type[, prolog[, epilog]]])``
```

Comment:
This change should probably be made in the commit titled `Remove OPTS argument from FUN.desc()`.